bench/db_bench.ml

```diff
@@ -35,6 +35,8 @@ let entry_size = key_size + value_size
 
 let random = Array.make nb_entries ("", "")
 
+let json_output = ref []
```

Comment:
It's generally considered bad practice to have global state like this. Perhaps you could have each benchmark return its results directly, then `List.map` rather than `List.iter` to get those results.